### PR TITLE
added Theme file

### PR DESCRIPTION
**Pull request by JustynaPawlik**

### CODE REVIEW

**Review by MCesarczyk — CHANGES_REQUESTED** [review]
Please, link theme by adding ThemeProvider to index.js. Adding Normalize library would be good idea too :)

File: src/index.js [hunk 1]
```diff
@@ -3,9 +3,16 @@ import ReactDOM from 'react-dom';
...
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { Normalize } from 'styled-normalize';
+import { GlobalStyle } from './GlobalStyle';
+import { ThemeProvider } from 'styled-components';
+import { theme } from './theme';
 
 ReactDOM.render(
   <React.StrictMode>
+    <ThemeProvider theme={theme}></ThemeProvider>
```
Comment (MCesarczyk) [inline src/index.js:13]: Theme provider should wrap around whole application to work properly

**Comment by JustynaPawlik** [comment]
I've just added ThemeProvider, Normalize and GlobalStyle in index.js

File: src/theme.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const theme = {
+    color: {
+    white: "#FFFFFF",
+    black: "#000000",
+},
+breakpoint: {
+    mobile : 767
```
Comment (MCesarczyk) [inline src/theme.js:7]: value of object properties should be in quotes, colon on the end of the line would help to avoid possible errors

**Review by MCesarczyk — CHANGES_REQUESTED** [review]
Well done, just a little adjustments required

File: src/index.js (hunk 1 above)
Comment (JustynaPawlik) [reply src/index.js]: you're right. I've changed it.

File: src/theme.js (hunk 2 above)
Comment (MCesarczyk) [reply src/theme.js]: sorry, I wrote "colon", but meant "comma" ;)

Comment (JustynaPawlik) [reply src/theme.js]: added comma to breakpoint

Comment (JustynaPawlik) [reply src/theme.js]: and added quotation marks in breakpoint

**Review by MCesarczyk — APPROVED** [review]
Great job, thanks 👍

**Pull request merged**